tests/framework-tests/preferences/html/Enactors-test.html

```diff
@@ -88,7 +88,7 @@ <h2 id="qunit-userAgent"></h2>
         </div>
 
         <div class="flc-tableOfContents">
-            <div class="flc-toc-tocContainer"></div>
+            <div class="flc-toc-tocContainer" style="display: none"></div>
```

Comment:
Add a comment here explaining that this attribute is explicitly a test for FLUID-6696

Reply:
I've added comments to both the test html and the enactor code.